### DISCUSSION TITLE
Watchtower Portainer Stack Issue: Cannot Connect to the Docker Daemon

### DISCUSSION THREAD

**Discussion by logabog** [containrrr/watchtower#1583] [Q&A]
Hello All,

I am new to Docker/Portainer and trying to deploy Watchtower as a stack in Portainer. My Portainer Stack is configured like so:

version: "3"
services:
  watchtower:
    image: containrrr/watchtower:latest
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock -v
    labels:
      - "com.centurylinklabs.watchtower.enable=true"
    restart: unless-stopped

![image](https://user-images.githubusercontent.com/97701974/223750980-17a260b6-9014-4c4a-a29f-5a9362fddfc3.png)

This yields the following error in the Portainer logs for the container:
![image](https://user-images.githubusercontent.com/97701974/223751744-7d6ce458-9abc-4a58-9a95-912bc172f44b.png)

I'm also not able to console in the machine from portainer and get this error when trying to do so:

![image](https://user-images.githubusercontent.com/97701974/223751631-3d1bec19-7312-4ce0-a158-fe3230abda8d.png)

I can connect to my other containers just fine and all is working in my Rasberry Pi Docker environment except Watchtower.

Could anyone please help in pointing me in the right direction for a solution?

**Comment by ewoks** [comment]
Does this still persist? not sure what is `-v` in your `/var/run/docker.sock` mapping, but that should be enough
Did restart host machine help?
